README.md

```diff
@@ -108,14 +130,14 @@ cran_packages: [ metricminer, ottrpal ]
 cran_googlesheet:
 ```
 
-### Setting up GitHub 
+### Setting up GitHub
 
 At this point you should already have your GitHub authorization set up for your metricminer dashboard by having [followed the instructions above.](#setting-up-your-dashboard-repository).
 
 - [ ] Make sure that `refresh-github` is set to "yes".
-- [ ] Specify the names of the repositories you'd like to collect data from in `github_repos`. Make sure it includes the `owner/repository` e.g. `fhdsl/metricminer` not just `metricminer`. Commas need to separate the repositories. 
+- [ ] Specify the names of the repositories you'd like to collect data from in `github_repos`. Make sure it includes the `owner/repository` e.g. `fhdsl/metricminer` not just `metricminer`. Commas need to separate the repositories. Delete the example repositories we put there.
```

Comment:
Do the repos you're collecting from also need a `GH_PAT`? Do you need to be in the organization to get data about their repository?

Reply:
Oh public vs private repo. Any considerations there?

Reply:
Hmmmm.... I'm not sure if it matters if its public or private.... something we should test.

Reply:
Made this an issue for a future testing #14